packages/ethereum-contracts/.solcover.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
         // we override the functions in child contracts
         "apps/SuperAppBase.sol",
         "apps/SuperAppBaseCFA.sol",
+        "apps/SuperfluidLoaderLibrary.sol",
```

Comment:
It's still not ignoring the file. Does the path need to be `apps/SuperfluidLoaderLibrary.sol`?

Reply:
hmm, that is weird, the path is fine as is, I am guessing this is coming from the foundry coverage.
we have a coverage report generated by hardhat which uses the `.solcover.js` file and we also have one generated by foundry which we are currently cleaning up ad-hoc.

To fix it, you need to modify `tasks/coverage-cleanup.sh`, you can add: 'packages/ethereum-contracts/contracts/apps/SuperfluidLoaderLibrary.sol' to the line where we remove stuff:
`lcov -r lcov.info 'packages/ethereum-contracts/contracts/mocks/*' 'packages/...' 'packages/...'`

Reply:
didn't know that I could go in and edit it myself, but seems like I can, went ahead and made the changes  :p

Reply:
Ohh thanks 👍